ui/pages/confirmations/components/confirm/info/hooks/use-token-values.ts

```diff
@@ -0,0 +1,54 @@
+import { TransactionMeta } from '@metamask/transaction-controller';
+import { useMemo } from 'react';
+import { calcTokenAmount } from '../../../../../../../shared/lib/transactions-controller-utils';
+import { toChecksumHexAddress } from '../../../../../../../shared/modules/hexstring-utils';
+import useTokenExchangeRate from '../../../../../../components/app/currency-input/hooks/useTokenExchangeRate';
+import { useFiatFormatter } from '../../../../../../hooks/useFiatFormatter';
+import { useTokenTracker } from '../../../../../../hooks/useTokenTracker';
+import { SelectedToken } from '../shared/selected-token';
+
+export const useTokenValues = (
```

Comment:
Is the target of this PR wrong as all these changes were also in the token send PR?

Reply:
should be up to date now